src/cli.ts

```diff
@@ -49,6 +49,13 @@ const argv = yargs(hideBin(process.argv))
       default: '',
       type: 'string',
     },
+    j: {
```

Comment:
I think we can have much more flexibility by using kql, using  jobId will restrict the use cases and tying us to that env variable used in perf tests.

Reply:
so in our case the query will be `labels.testJobId: "..." and labels.journeyName:"login"` and we can pretty much query everything that we can query in kql another example use case might be filtering out based on journeyname. Currently by relying only test job id we are collecting all journeys (login,ecommerce dashboard etc.) .

Reply:
> Currently by relying only test job id we are collecting all journeys (login,ecommerce dashboard etc.)

This is a valid point. We can start using two parameters: `jobId`, `journeyName` instead of a query. @dmlemeshko WDYT?

Reply:
Pushed 8e267ac3561562bc370f50b17168fea520ea6a28 with those 2